### PR TITLE
Pal 1092 generate coderelease and tag

### PR DESCRIPTION
**Pull request by dev930018**

### CODE REVIEW

File: example-runner/pom.xml [hunk 1]
```diff
@@ -49,11 +49,15 @@
     </scm>
 
     <properties>
+        <jackson.version>2.11.0</jackson.version>
```
Comment (m78233) [inline example-runner/pom.xml:52]: Looks like this was not meant to be changed

Comment (dev930018) [reply example-runner/pom.xml:52]: Just moved to the top in line with the style used in other poms, dependency versions first, then dockerfile things last

File: example-runner/src/main/resources/values.yaml [hunk 2]
```diff
@@ -21,9 +21,7 @@ service:
 image:
   name: @artifactId@
   base: @dockerfile.base.tag@
-  tag: @revision@-@git.commit.id.abbrev@
   pullPolicy: IfNotPresent
-  codeRelease: @project.parent.version@
```
Comment (m78233) [inline example-runner/src/main/resources/values.yaml:26]: deployment-jvm module  needs the same refactoring to remove codeRelease and tag as was done here in example-runner.

**Review by m78233 — CHANGES_REQUESTED** [review]
Minor issues, not much else.  I think it will be ok with the changes in PAL-1081, but it might b e worth checking.

**Review by m78233 — APPROVED** [review]
Looks good

File: NOTICES.md [hunk 3]
```diff
@@ -6,11 +6,11 @@ List of third-party dependencies grouped by their license type
...
-* GCHQ Palisade - Resource Service ([uk.gov.gchq.palisade:resource-service:0.5.1-SNAPSHOT](https://github.com/gchq/Palisade-services/tree/develop/resource-service))
-* GCHQ Palisade - User Service ([uk.gov.gchq.palisade:user-service:0.5.1-SNAPSHOT](https://github.com/gchq/Palisade-services/tree/develop/user-service))
+* GCHQ Palisade - Akka Client ([uk.gov.gchq.palisade:client-akka:0.5.0-SNAPSHOT](https://github.com/gchq/Palisade-clients/tree/develop/client-akka))
+* GCHQ Palisade Common Library ([uk.gov.gchq.palisade:common:0.5.0-SNAPSHOT](https://github.com/gchq/Palisade-common))
+* GCHQ Palisade - Example Library ([uk.gov.gchq.palisade:example-library:0.5.0-SNAPSHOT](https://github.com/gchq/Palisade-examples/tree/develop/example-library))
+* GCHQ Palisade - Resource Service ([uk.gov.gchq.palisade:resource-service:0.5.0-SNAPSHOT](https://github.com/gchq/Palisade-services/tree/develop/resource-service))
+* GCHQ Palisade - User Service ([uk.gov.gchq.palisade:user-service:0.5.0-SNAPSHOT](https://github.com/gchq/Palisade-services/tree/develop/user-service))
```
Comment (developer01189998819991197253) [inline NOTICES.md:13]: Dont we want to build against 0.5.1?

**Review by ac74475 — CHANGES_REQUESTED** [review]
Doesnt work for the release profile

**Review by ac74475 — DISMISSED** [review]
it now works for the release profiles

**Review by developer01189998819991197253 — APPROVED** [review]

**Pull request merged**